talentsearch/resources/js/talentSearchRoutes.ts

```diff
@@ -1,5 +1,3 @@
-import { BASE_URL_DIR } from "./talentSearchConstants";
-
-export const homePath = (): string => `${BASE_URL_DIR}`;
+export const homePath = (): string => `${process.env.TALENTSEARCH_APP_DIR}`;
```

Comment:
I'd like to keep any reference to `process.env` in the talentSearchConstants.ts file. That makes it easy to just look at one file, to see all the environment variables this project relies on (even if that is currently just a single variable).